### PR TITLE
feat(docs-infra): layout ui polish

### PR DESCRIPTION
**Pull request by sjtrimble**
Adds spacing to callout text area, adds vertical spacing to lists and code tabs, css auto-reformat for code legibility (need to do more of this later).

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
Inconsistent layout spacing issues

Issue Number: #30880 

## What is the new behavior?
Adds spacing where appropriate for better legibility and consistency within app

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/src/styles/2-modules/_images.scss [hunk 1]
```diff
@@ -37,7 +37,7 @@
     padding: 20px;
     display: inline-block;
     box-shadow: 2px 2px 5px 0 rgba(0, 0, 0, .2);
-    margin: 0 0 14px 0;
+    margin: 16px 0;
```
Comment (gkalpak) [inline aio/src/styles/2-modules/_images.scss]: I am not a big fan of the left margin, but 🤷‍♂

**Review by gkalpak — APPROVED** [review]

File: aio/src/styles/2-modules/_images.scss (hunk 1 above)
Comment (kapunahelewong) [reply aio/src/styles/2-modules/_images.scss]: @gkalpak that's only top and bottom that the 16px is referring to.

Comment (gkalpak) [reply aio/src/styles/2-modules/_images.scss]: Aha, thx!
(For some reason I read `margin: 16px;` 🤦‍♂ I blame the green highlighting 😛)

**Comment by brandonroberts** [comment]
@sjtrimble will you rebase on master to fix the merge conflicts?

**Comment by sjtrimble** [comment]
@brandonroberts just rebased

**Comment by brandonroberts** [comment]
Caretaker note: The Codefresh failure is unrelated

**Pull request closed without merging**